apps/web/README.md

```
@@ -1,197 +1,142 @@
-* `fix`: A bug fix
-* `docs`: Documentation only changes
-* `style`: Changes that do not affect the meaning of the code
-* `refactor`: A code change that neither fixes a bug nor adds a feature
-* `chore`: Changes to the build process or auxiliary tools
+The application strictly enforces **Feature-Sliced Design (FSD)** across client and server boundaries:
 
-**Example:**
-
-```bash
-git commit -m "feat(repo): add copy-paste detection via jscpd"
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)